sidechain/top-pool-executor/src/call_operator.rs

```diff
@@ -43,7 +43,7 @@ pub trait TopPoolCallOperator<
 	fn get_trusted_calls(
```

Comment:
```suggestion
	fn get_trusted_operations(
```

Reply:
Okay to keep `calls` for now? (as with the other instances of these suggestions)

Reply:
Yes, ofc.